docs/user-guides/hunter-user/git-submodule.rst

```diff
@@ -107,6 +108,19 @@ Run tests to see changes:
   1:   plum-v2 x 2
   1:   pear x 1
 
+Use subdirectory of submodule
+=============================
+
+To instruct hunter to archive a subdirectory of the Git submodule add the keyword ``HUNTER_SUBMODULE_SOURCE_SUBDIR`` to the cmake arguments:
+
+.. code-block:: cmake
+
+  # cmake/Hunter/config.cmake
+  hunter_config(fruits GIT_SUBMODULE "3rdParty/fruits"
+    CMAKE_ARGS HUNTER_SUBMODULE_SOURCE_SUBDIR="app")
```

Comment:
Should be `HUNTER_SUBMODULE_SOURCE_SUBDIR=app`  or `"HUNTER_SUBMODULE_SOURCE_SUBDIR=app"`. It may work in the current case but in general we should avoid it (related: http://cgold.readthedocs.io/en/latest/tutorials/variables/regular.html#quotes).